docs/data/data-core-editor.md

```diff
@@ -1253,6 +1253,19 @@ Returns whether or not the user has the unfiltered_html capability.
 
```

Comment:
What are these changes? Are they related to this PR?

Reply:
Not at all. This is sort of automated with `npm run docs:build` ...
It added it in addition to the manifest file changes. Someone forgot to do it when updating those methods ...

Reply:
Ah... Might be better to merge separately then. I'll have a look in a bit.